src/sentry/models/grouphistory.py

```diff
@@ -74,7 +75,11 @@ class GroupHistoryStatus:
     GroupHistoryStatus.DELETED_AND_DISCARDED,
 ]
 
-UNRESOLVED_STATUSES = (GroupHistoryStatus.UNRESOLVED, GroupHistoryStatus.REGRESSED)
+UNRESOLVED_STATUSES = (
```

Comment:
What is the purpose of this list of status?
Should `Escalating` be part of it?

Reply:
I think its used for state transition changes. This should probably be in there if `Escalating` is completely released in the wild and ready for use.f

Reply:
After our conversation yesterday, ONGOING will be a substatus and keep UNRESOLVED as the status, thus, we should not add ONGOING in here.